packages/app/RNFBApp.podspec

```diff
@@ -28,11 +28,4 @@ Pod::Spec.new do |s|
 
   # Firebase dependencies
   s.dependency          'Firebase/CoreOnly', firebase_sdk_version
-
-  if defined?($RNFirebaseAsStaticFramework)
```

Comment:
I'm not sure this is the same thing as the Firebase SDK 'linkage' change, this is for the RNFB podspec itself, rather than the Firebase SDKs, different things?

(same in other podspecs)

Reply:
Mmm - will look more closely and (likely) revert. This part is voodoo to me, never had it as a personal need so never learned about it

Reply:
Agreed on this part, and I reverted the related items, after reading more. I altered the commit message to be more equivocal:

```
   This also may give you options on how you link firebase-ios-sdk. If you have special linking needs, read:
      - https://firebase.google.com/docs/ios/link-firebase-static-dynamic
```